local-infrastructure/README.md

```diff
@@ -31,14 +31,13 @@ docker-compose build
 #### Stand up services
 
 ```sh
-docker-compose up -d postgres keycloak
+docker-compose up -d
 ```
-Note that the node_migrate service does not continually run on up.  We run it only to populate the database. See below.
+Note that keycloak and postgress will run indefinitely, while node_migrate performs migrations and stops.
```

Comment:
There should be a mention somewhere in this readme to consider using the run.sh script as a quickstart.

Reply:
Done!